scripts/docs-versions.sh

```diff
@@ -7,6 +7,6 @@ for ver in $(git ls-remote --heads "${1:-.}" | grep -P -o '(?<=refs/heads/)[0-9]
   printf '{"version": "%s", "title": "%s", "aliases": [%s]}, ' "$ver" "$ver" "$latest"
   latest=''
 done
-printf '{"version": "master", "title": "nightly", "aliases": []}'
+printf '{"version": "master", "title": "nightly", "aliases": ["latest"]}'
```

Comment:
Why is master latest?

Reply:
To not show that it's "(outdated)"

Reply:
But it's also not the latest stable release. There *should* be a warning that you're reading unreleased documentation.
The message wording "You're not viewing the latest version" fits well for that. It fits for every version that is not the latest stable release.

Reply:
OK